generator/src/test/java/org/stjs/generator/writer/namespace/NamespaceGeneratorTest.java

```diff
@@ -111,10 +111,7 @@ public void testNamespaceDefinedInConfiguration() {
 
 		assertCodeContains(Namespace13_generator_configuration.class, "" +
 						"stjs.ns(\"my.own.namespace\");\n" +
-						"my.own.namespace.Namespace13_generator_configuration = function(outerClass$0) {\n" +
-						"    this._outerClass$0 = outerClass$0;\n" +
-						"};\n" +
-						"my.own.namespace.Namespace13_generator_configuration = stjs.extend(my.own.namespace.Namespace13_generator_configuration, null, [], function(constructor, prototype) {",
+						"my.own.namespace.Namespace13_generator_configuration = function() {}",

```

Comment:
Looks like you changed the expected result of the test to allow the test to pass, but it is quite a change, are you sure about this one?